eras/alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -108,6 +108,7 @@ library
 library testlib
   exposed-modules:
     Test.Cardano.Ledger.Alonzo.Arbitrary
+    Test.Cardano.Ledger.Alonzo.Binary
```

Comment:
Since there are other modules below `Binary`, and this isn't a re-exporting module for them, I think it might be better to place this new module below `Binary` with a name that reflects its specialized purpose. Perhaps `Test.Cardano.Ledger.Alonzo.Binary.TxSize` or something like that.

The same applies for the Shelley one, of course.

Reply:
We already have this for example: `Imp` module  +  `Imp.x.y` , even `Binary` + `Binary.x.y` in core.